### PR TITLE
Gutenboarding: don't translate

### PR DESCRIPTION
**Pull request by simison**
Avoid sending strings for the translation tool in gutenboarding section by re-exporting `@wordpress/i18n` translation function with `NO` prefix: `__( 'Hello, world!' )` -> `NO__( 'Hello, world!' )`.

#### Testing instructions

* Builds and works as expected (visit [`/gutenboarding`](https://calypso.live/gutenboarding?branch=update/gutenboarding-dont-translate) and look for "translated" strings).
* Strings should not be present in parsed translations.

### CODE REVIEW

File: client/gutenboarding/devtools.ts [hunk 1]
```diff
@@ -18,3 +18,10 @@ export const wpDataDebugMiddleware: PageJS.Callback = ( context, next ) => {
 	}
 	next();
 };
+
+/**
+ * Function to use instead of `__()` from `@wordpress/i18n`.
+ *
+ * This is to avoid sending strings for translation tool too early.
```
Comment (simison) [inline client/gutenboarding/devtools.ts]: TODO; needs a valid JSdoc says linter

Comment (sirreal) [reply client/gutenboarding/devtools.ts]: We need to update or drop the jsdoc lint rules.

Comment (sirreal) [reply client/gutenboarding/devtools.ts]: If you add a space after the comment block, the lint issue may go away.

We'll want all the `_x`, `_n`, etc. family I think.

File: client/gutenboarding/onboarding-block/index.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __ as NO__ } from '@wordpress/i18n';
```
Comment (sirreal) [inline client/gutenboarding/onboarding-block/index.ts]: Use `devtools`?

File: client/gutenboarding/devtools.ts (hunk 1 above)
Comment (simison) [reply client/gutenboarding/devtools.ts]: Space helped 👍

**Comment by simison** [comment]
Added rest of the functions, good for another look!

**Comment by sirreal** [comment]
I rebased this and reworked it so we reexport the i18n functions with a different name.

**Comment by sirreal** [comment]
Recent `master` runs of `lint-and-translate` job I can see the following in the artifacts:

```
#: client/gutenboarding/onboarding-block/edit.tsx:25
msgid "with a blog."
msgstr ""
```

Doesn't seem to be present in the artifacts from this branch.

**Comment by ockham** [comment]
Bit late to the party :sweat_smile: But can't we just `import { __ as NO__ } from '@wordpress/i18n'`? I don't think our string extraction is smart enough to look for changed import names.

Just a bit skeptical about the `devtools` thing -- we had an i18n wrapper like that in Jetpack for a while, and IIRC, getting rid of it wasn't the most straightforward thing in the world :sweat_smile:

**Comment by sirreal** [comment]
> Bit late to the party 😅 But can't we just `import { __ as NO__ } from '@wordpress/i18n'`? I don't think our string extraction is smart enough to look for changed import names.
> 
> Just a bit skeptical about the `devtools` thing -- we had an i18n wrapper like that in Jetpack for a while, and IIRC, getting rid of it wasn't the most straightforward thing in the world 😅

Sounds good, let's do that 👍 

@razvanpapadopol will you handle that change? Then we should be all set to merge.

**Pull request merged**